score-jClouds/src/main/java/io/cloudslang/content/jclouds/entities/DiskTypes.java

```diff
@@ -0,0 +1,36 @@
+package io.cloudslang.content.jclouds.entities;
+
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Created by Mihai Tusa.
+ * 2/19/2016.
+ */
+public enum DiskTypes {
+    GENERAL_PURPOSE_SSD("gp2"),
+    PROVISIONED_IOPS_SSD("io1"),
+    MAGNETIC("standard");
+
+    private final String value;
+
+    DiskTypes(String value) {
+        this.value = value;
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+    public static String getDiskType(String input) {
```

Comment:
@gituser173  It seems that I no longer need this... so I will remove it. Good catch!